GetFed/GetFed/Views/FoodResultTableViewCell.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  FoodResultTableViewCell.swift
+//  GetFed
+//
+//  Created by Britney Smith on 11/21/18.
+//  Copyright © 2018 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+class FoodResultTableViewCell: UITableViewCell {
+
+    // MARK - Outlets
+    
+    @IBOutlet weak var foodLabel: UILabel!
+    @IBOutlet weak var brandLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
```

Comment:
If all the method does is call `super`, you can delete it.

Reply:
Thanks! Deleted `awakeFromNib`. I still might do more with `setSelected`, though.